config/webhook/manifests.yaml

```diff
@@ -15,6 +15,7 @@ webhooks:
       namespace: system
       path: /validate-agent-k8s-elastic-co-v1alpha1-agent
   failurePolicy: Ignore
+  matchPolicy: Exact
```

Comment:
What happens on k8s versions < 1.15 where I believe this property did not exist yet?

Reply:
Good catch. I should have checked the versions where it is supported.